### PR TITLE
JCSP-391: DC, Cloud: Context operation on board page

### PR DESCRIPTION
**Pull request by ysemenkov**

### CODE REVIEW

File: custom-vu/src/main/kotlin/jces1209/vu/action/ViewBoard.kt [hunk 1]
```diff
@@ -9,18 +9,21 @@ import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
...
+import org.openqa.selenium.WebDriver
 
 class ViewBoard(
+    private val driver: WebDriver,
     private val meter: ActionMeter,
     private val boardsMemory: SeededMemory<BoardPage>,
     private val issueKeyMemory: IssueKeyMemory,
     private val random: SeededRandom,
     private val viewIssueProbability: Float,
-    private val jiraTips: JiraTips
```
Comment (dagguh) [inline custom-vu/src/main/kotlin/jces1209/vu/action/ViewBoard.kt:24]: It's inlined later on, ok 👍

File: custom-vu/src/main/kotlin/jces1209/vu/MeasureType.kt [hunk 2]
```diff
@@ -5,6 +5,9 @@ import com.atlassian.performance.tools.jiraactions.api.ActionType
 class MeasureType {
 
     companion object {
+        @JvmField
+        val CONTEXT_OPERATION_BOARD = ActionType("Context operation (Board)") { Unit }
```
Comment (dagguh) [inline custom-vu/src/main/kotlin/jces1209/vu/MeasureType.kt:9]: Adding it to the top, so it doesn't conflict with the bottom additions in other PRs, smart 🧠

File: custom-vu/src/main/kotlin/jces1209/vu/action/ViewBoard.kt [hunk 3]
```diff
@@ -49,16 +52,31 @@ class ViewBoard(
...
                 logger.debug("It requires some issues on board to test preview issue")
             } else {
+                val jiraTips = JiraTips(driver)
                 jiraTips.closeTips()
                 meter.measure(MeasureType.ISSUE_PREVIEW_BOARD) {
                     meter.measure(ActionType("Preview issue ($boardType board)") { Unit }) {
                         board.previewIssue()
                     }
                 }
+                board.closePreviewIssue()
```
Comment (dagguh) [inline custom-vu/src/main/kotlin/jces1209/vu/action/ViewBoard.kt]: Just a code-style nit. I can no longer fit the entire method on a single screen.

**Review by dagguh — DISMISSED** [review]

**Review by dagguh — DISMISSED** [review]

**Comment by ysemenkov** [comment]
I've just fixed conflicts

**Review by NataliiaShavshyna — APPROVED** [review]

**Pull request merged**